src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/script-approval.js

```diff
@@ -0,0 +1,236 @@
+        Array.from(document.getElementById('approvedClasspathEntries').children).forEach(function(e){e.remove()});
+        document.getElementById('approvedClasspathEntries').style.display = 'none';
+        document.getElementById('approvedClasspathEntries-clear').style.display = 'none';
+    } else {
+        document.getElementById('approvedClasspathEntries-none').style.display = 'none';
+        Array.from(document.getElementById('approvedClasspathEntries').children).forEach(function(e){e.remove()});
+        approvedClasspathEntries.forEach(function(e) {
+            var block = document.createElement('p');
+            block.setAttribute('id', 'acp-' + e.hash);
+            var deleteButton = document.createElement('button');
+            deleteButton.setAttribute('class', 'delete');
+            deleteButton.setAttribute('hash', e.hash);
+            deleteButton.textContent = 'Delete';
+            deleteButton.addEventListener('click', function() {
+                if (confirm('Really delete this approved classpath entry? Any existing scripts using it will need to be rerun and the entry reapproved.')) {
+                    denyApprovedClasspathEntry(this.getAttribute('hash'));
+                }
+            });
+            block.appendChild(deleteButton);
+            var code = document.createElement('code');
+            code.setAttribute('title', e.hash);
+            code.textContent = e.path;
+            block.appendChild(code);
+
+            document.getElementById('approvedClasspathEntries').appendChild(block);
+        });
+        document.getElementById('approvedClasspathEntries').style.display = '';
+        document.getElementById('approvedClasspathEntries-clear').style.display = '';
+    }
+}
+
+function renderClasspaths(r) {
+    renderPendingClasspathEntries(r.responseObject()[0]);
+    renderApprovedClasspathEntries(r.responseObject()[1]);
+}
+
+function approveClasspathEntry(hash) {
+    mgr.approveClasspathEntry(hash, function(r) {
+        renderClasspaths(r);
+    });
+}
+function denyClasspathEntry(hash) {
+    mgr.denyClasspathEntry(hash, function(r) {
+        renderClasspaths(r);
+    });
+}
+function denyApprovedClasspathEntry(hash) {
+    mgr.denyApprovedClasspathEntry(hash, function(r) {
+        renderClasspaths(r);
+    });
+}
+function clearApprovedClasspathEntries() {
+    mgr.clearApprovedClasspathEntries(function(r) {
+        renderClasspaths(r);
+    });
+}
+
+document.addEventListener('DOMContentLoaded', function() {
+    mgr.getClasspathRenderInfo(function(r) {
+        renderClasspaths(r);
+    });
+
+    const approveScriptButtons = document.querySelectorAll('.ps-context button.approve');
+    approveScriptButtons.forEach(function (button) {
+        button.addEventListener('click', function () {
+            approveScript(button.dataset.hash);
+        });
+    });
+
+    const denyScriptButtons = document.querySelectorAll('.ps-context button.deny');
```

Comment:
This element is always displayed, so no need for an `if` statement on the next line.